src/components/home/ProjectSection/Project.tsx

```diff
@@ -7,6 +7,9 @@ type Props = {
     desc: string;
     imageUrl: string;
     projectUrl: string;
+    projectId: string; 
+    bodyId: string; 
+    imgId: string
```

Comment:
the body id and img id can all be derived from project id. With image if, we can defiend the project image url in CSS, so imageUrl is not needed.

Reply:
e.g.: <Card.Body id={"body_"+this.propsprojectId}>

Reply:
That's very helpful, thanks for the suggestion!  Changes have been done.